### PR TITLE
Feature/83 create account UI beta

### PR DESCRIPTION
**Pull request by Riae**
I can't seem to figure out how to make Flexbox work with RN Paper. I've scoured the internet for a bit, but there's nothing on this. Any ideas?

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -2,13 +2,22 @@ import { StatusBar } from 'expo-status-bar';
...
+
 export default function App() {
   return (
     <PaperProvider>
       <View style={styles.container}>
-        <Text>Open up App.tsx to start working on your app! </Text>
-        <TButton>test</TButton>
+
+      <Ttextinput>Username</Ttextinput>
+      <Ttextinput>Password</Ttextinput>
```
Comment (theblockstalk) [inline App.tsx:14]: Password should be adjusted to not show characters as typed, instead ***

File: src/components/Tbutton.tsx [hunk 2]
```diff
@@ -7,6 +7,16 @@ type Props = {
 
 export default function TButton({ children }: Props) {
   return (
-    <Button mode="contained">{children}</Button>
+    <Button 
+    mode="contained"
+    contentStyle={{
+      width: 340,
```
Comment (theblockstalk) [inline src/components/Tbutton.tsx:13]: this styling should be where the button is used, not in the generic component. this should be in the CreateAccountScreeen (App.tsx)

File: src/components/Tbutton.tsx [hunk 3]
```diff
@@ -7,6 +7,16 @@ type Props = {
 
 export default function TButton({ children }: Props) {
   return (
-    <Button mode="contained">{children}</Button>
+    <Button 
+    mode="contained"
+    contentStyle={{
+      width: 340,
+      height: 50,
+      flex: 1,
```
Comment (Riae) [inline src/components/Tbutton.tsx:15]: Can't seem to make flexbox work with RN Paper on this, although docs say this should work.

File: App.tsx [hunk 4]
```diff
@@ -2,13 +2,22 @@ import { StatusBar } from 'expo-status-bar';
...
+
+
 export default function App() {
   return (
     <PaperProvider>
       <View style={styles.container}>
-        <Text>Open up App.tsx to start working on your app! </Text>
-        <TButton>test</TButton>
+
+      <Ttextinput>Username</Ttextinput>
```
Comment (Riae) [inline App.tsx:13]: Or should the layout be done here?

File: src/components/TPasswordInput.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import { TextInput } from 'react-native-paper';
+
+type Props = {
+  children: JSX.Element | JSX.Element[] | string;
+};
+
+export default function TPasswordInput({ children }: Props) {
+  return (
+    <TextInput
+      rows={1}
```
Comment (theblockstalk) [inline src/components/TPasswordInput.tsx:11]: why rows={1}? should this be for the user to specify?

File: App.tsx [hunk 6]
```diff
@@ -1,19 +1,49 @@
...
+        </View>
+
         <StatusBar style="auto" />
+
       </View>
     </PaperProvider>
   );
 }
 
+const colors = {
```
Comment (theblockstalk) [inline App.tsx:41]: remove. we will do later with Theme
https://callstack.github.io/react-native-paper/theming.html

File: App.tsx [hunk 7]
```diff
@@ -22,3 +52,17 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
   },
 });
+const WidthHeight = StyleSheet.create({
```
Comment (theblockstalk) [inline App.tsx:55]: please use more consistent variable names and capital conventions (camel case) for the stylesheets
e.g. createAccountButtonStyle
formViewStyle's

File: src/components/TPasswordInput.tsx (hunk 5 above)
Comment (Riae) [reply src/components/TPasswordInput.tsx:11]: Misread the function of the prop, it is not needed after all. Removed.

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**